### ISSUE TITLE
Using definition results in unknown type

### ISSUE THREAD

**Issue by rlau89** [ThomasAribart/json-schema-to-ts#49]
Version: 1.6.4
--- 
Take the following simple schema

```
export const simpleSchema = {
  $schema: 'https://json-schema.org/draft-2020-12/schema',
  type: 'object',
  properties: {
    exampleObject: {
      type: 'object',
      properties: {
        name: {
          $ref: '#/definitions/example'
        }
      },
      required: ['name'],
      additionalProperties: false
    }
  },
  required: [
    'exampleObject'
  ],
  additionalProperties: false,
  definitions: {
    example: {
      type: 'string'
    }
  }
} as const
```

When converting this to TS using FromSchema results in the name property being an unknown type. 
```
import { simpleSchema } from './simpleSchema'
type SimpleSchema = FromSchema<typeof simpleSchema>
```

![image](https://user-images.githubusercontent.com/28444541/146263158-271033a6-d792-47e5-baf5-a008d8777168.png)

Hoping someone can shed some light on this for me. Hoping I'm making a simple mistake somewhere

**Comment by ThomasAribart** [comment]
Hello @rlau89 !

You're not doing any mistake, the `definitions` keyword is not supported by `FromSchema` for now. It's my next goal on this lib and should be done by the beginning of 2022.

**Comment by rlau89** [comment]
Thanks for getting back so quick. After posting I noticed the README stating it was coming and this related [enhancement](https://github.com/ThomasAribart/json-schema-to-ts/issues/13).

I have moved to creating TS enum's and getting `Object.keys` in places to provide reusability. Also combining schemas by importing them which is really useful. With this approach I don't necessarily need the definition support.
